schema/schema.go

```diff
@@ -72,6 +73,19 @@ func Validate(config map[string]interface{}) error {
 	return nil
 }
 
+// removeSecretsContentProperty removes the content property from secrets
+// we add the content key here loader/environment.go:66
+func removeSecretsContentProperty(config map[string]interface{}) map[string]interface{} {
```

Comment:
@ndeloof here is an equivalent ) at least tmp solution

Reply:
but we loose value doing so

Reply:
we can copy for validation

Reply:
pushed changes. now it clones the map and validate